js/coinex.js

```diff
@@ -2730,9 +2730,10 @@ module.exports = class coinex extends Exchange {
             const bracket = item[j];
             const leverage = this.safeInteger (bracket, 1);
             const maxNotional = this.safeNumber (bracket, 0);
+            const currency = (market['linear']) ? market['base'] : market['quote'];
```

Comment:
Did you check and confirm that this is the case for all markets?

Also, you can remove the () around market['linear']

Reply:
Yeah I checked and this is the case for all markets